star-ros2/src/star_gateway_bridge/include/star_gateway_bridge/message_converter.hpp

```
@@ -152,10 +154,7 @@ class MessageConverter
    * @param value Value to validate
    * @return true if value is finite (not NaN, not infinity)
    */
-  static bool is_valid_double(double value)
-  {
-    return std::isfinite(value);
-  }
+  static bool is_valid_double(double value) {return std::isfinite(value);}
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)